src/behaviors/focusTrap.ts

```diff
@@ -83,6 +84,7 @@ export function focusTrap(
         } else {
           const toFocus = initialFocus && container.contains(initialFocus) ? initialFocus : getFocusableChild(container)
           if (toFocus) {
+            prepareForFocusWithoutMouse()
```

Comment:
Is this going to be affected by https://github.com/primer/components/pull/1157?

Reply:
This was a hacked fix before #1157.  I'll get master merged in and remove this.